ui/components/ui/typography/typography.js

```diff
@@ -46,6 +46,7 @@ export const ValidTags = [
   'span',
   'strong',
   'ul',
+  'label',
```

Comment:
Allows for the root html element of the `Typography` component to be rendered as a `<label />`